src/SprykerEco/Zed/Adyen/Dependency/Facade/AdyenToSalesFacadeBridge.php

```diff
@@ -29,7 +29,7 @@ public function __construct($salesFacade)
      *
      * @return \Generated\Shared\Transfer\OrderTransfer
      */
-    public function getOrderByIdSalesOrder(int $idSalesOrder): OrderTransfer
```

Comment:
Why removing? Is this not an int?

Reply:
I suppose sniffer did it. In SalesFacade it looks like
`public function getOrderByIdSalesOrder($idSalesOrder)`

Reply:
Ther sniffer does not remove typehints. if they are already in place we might just want to keep them.

Reply:
I got a sniffer error that the bridge signature should be completely the same. That's why I've removed type hint to not have this error while committing the changes

Reply:
I see, ok.